packages/web-components/fast-components/src/tree-view/fixtures/tree-view.html

```diff
@@ -0,0 +1,138 @@
+<fast-design-system-provider use-defaults>
+
+    <h4>With items (don't render collapsed nodes)</h4>
+    <fast-tree-view render-collapsed-nodes="false">
+        <fast-tree-item>
+            Root item
+            <fast-tree-item slot="item">
```

Comment:
Can fast-tree-item set a slot on itself of "item" automatically? I believe we have this in place for the accordion.

Reply:
Wondering if this could be an issue to refactor this? I tried a couple things and it was fubar'd

Reply:
because this is slightly different than accordion-item in its recursive nature, this was not working for me. This may be something we can work out but I think it requires a fix to something outside of the scope of this PR, re-activate if you disagree, will discuss in discord as well